images/runtime/dotnetcore/Dockerfile.template

```diff
@@ -16,7 +16,8 @@ FROM %DOTNETCORE_BASE_IMAGE%
 # a version of `curl` that has known issues.
 # We manually update it here so we can still depend on the original images.
 # This command should be removed once support for deprecated .NET core images is halted.
-RUN apt update && apt install curl --yes
+RUN sed -i '/jessie-updates/d' /etc/apt/sources.list  # Now archived
+RUN apt-get update && apt-get install curl --yes
```

Comment:
@gpcastro Was there a reason behind using `apt-get`(like we usually do in other places) instead of `apt`? I made the change to `apt-get` as I was seeing the following warning, so just changed it:
`WARNING: apt does not have a stable CLI interface yet. Use with caution in scripts.`